storage/wdpost_run.go

```diff
@@ -659,6 +659,7 @@ func (s *WindowPoStScheduler) runPoStCycle(ctx context.Context, di dline.Info, t
 
 				if !bytes.Equal(checkRand, rand) {
```

Comment:
I'm trying to understand how this can happen.  The challenge epoch should be a constant and my limited understanding of beacon randomness is that the epoch to beacon mapping never changes.  Chain reorgs should also never change which randomness goes with which epoch. Is the last condition not being met because of the beacon bug we didn't fully fix in hyperdrive?

Reply:
> Is the last condition not being met because of the beacon bug we didn't fully fix in hyperdrive?

Yes.